### ISSUE TITLE
Ticket currencies

### ISSUE THREAD

**Issue by MysterAitch** [openSUSE/osem#2086]
**I'm submitting a ..**
- Bug Report

**Current behaviour:**
Only one currency is permitted per conference. When creating/editing tickets, the option to select alternative currencies is still visible/available.

**Expected correct behaviour:**
If the single currency per conference is to be enforced, then it should be configured on a per-conference rather than per-ticket basis. If currencies are shown on the ticket configuration page, then it should be a read-only/non-editable form field.

Alternatively, support for multiple currency tickets per conference should be available. 

**Steps to reproduce:**
Attempt to create a ticket with a currency that is not the same as existing tickets.

**Comment by vishaldeepak** [comment]
Interested in giving this a shot. Expected Correct behaviour has two ways, which one is  the preferred solution?

**Comment by differentreality** [comment]
This is marked as a good first issue for the option of disabling the ticket currency selection.

Enabling multiple currencies per conference (possibly with conversion) is a bigger issue.

So for the time being, I would suggest you merely disable the currency selection when the currency is already decided.

**Comment by ed1001** [comment]
Hiya, this would be my first ever contribution :) May I take this on?

**Comment by differentreality** [comment]
Sure @ed1001 go ahead :+1:

**Comment by Ana06** [comment]
I do not think we should support several currencies for a conference. OSEM is already complex enough, so I would prefer keeping things simple.

Regarding the single currency per conference, this shouldn't be managed in the tickets. As all the tickets should have the same currency, this should be an attribute of conference. So, the correct approach is in my opinion:

1. Add a currency attribute to conference
2. Allow to set this attribute in the Basics page of the conference (admin page)
3. create a data rake task to set this new attribute to the tickets currency (it should be the same in all the ticket but I would double check this in the rake task - also don't forget to document this rake task for the updates)
4. remove the currency attribute from the ticket

And this may not be a `good first issue`.
